test_conformance/extensions/cl_khr_command_buffer/command_buffer_printf.cpp

```diff
@@ -274,7 +274,7 @@ struct CommandBufferPrintfTest : public BasicCommandBufferTest
                                      &pattern[0], 0, nullptr, nullptr);
         test_error(error, "clEnqueueWriteBuffer failed");
 
-        cl_int offset[] = { 0, pattern.size() - 1 };
+        size_t offset[] = { 0, pattern.size() - 1 };
         error = clEnqueueWriteBuffer(queue, off_mem, CL_TRUE, 0, sizeof(offset),
                                      offset, 0, nullptr, nullptr);
         test_error(error, "clEnqueueWriteBuffer failed");
```

Comment:
off_mem is created for sizeof(cl_int) size so I think the enqueuewritebuffer to it using sizeof(size_t) can cause issues.

Reply:
This is now using `cl_uint` for the type of the array and an assert is added to ensure the casting is safe.